ontology/yaml/resources/HVAC/entity_types/SENSOR.yaml

```diff
@@ -120,3 +120,62 @@ SENSOR_FDPM:
     - local_air_temperature_sensor
     - battery_charge_percentage_sensor
     - signal_strength_percentage_sensor
+
+SENSOR_CO_CO2_ATM:
+  description: "Atmosphere CO and CO2 combined sensor with local temp and RH readings."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  uses:
+  - thirtysecondrolling_average_zone_air_co_concentration_sensor
+  - zone_air_co_concentration_sensor
+  - zone_air_co2_concentration_sensor
+  - local_air_relative_humidity_sensor
+  - local_air_temperature_sensor
+
+SENSOR_FORM_ATM:
+  description: "Atmosphere formaldehyde combined sensor with local temp and RH readings."
+  is_canonical: true
+  implements:
+  - /SENSOR
+  uses:
+  - zone_air_formaldehyde_concentration_sensor
+  - local_air_relative_humidity_sensor
+  - local_air_temperature_sensor
+
+SENSOR_OZONE_NO_ATM:
```

Comment:
why not NO2 if that's what we're measuring?

Reply:
oops. forgot to add the 2.